src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -138,6 +138,13 @@ export default class DataBrowser extends React.Component {
     if (this.props.disableKeyControls) {
       return;
     }
+    if(this.props.newObject){
```

Comment:
I believe if I click ESC when editting a cell, it will cancel the entire row, while it should only cancel the row I am currently editting, right?

Reply:
yes, It should not cancel the new row, when editing other rows. Just updated the pr. please let me know about the changes